src/rules/declaration-block-no-ignored-properties/__test__/index.js

```diff
@@ -0,0 +1,178 @@
+import testRule from "../../../testUtils/testRule"
+
+import rule, { ruleName, messages } from ".."
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+
+  accept: [ {
+    code: "a { display: inline; }",
+  }, {
+    code: "a { display: inline; color: red; }",
+  }, {
+    code: "a { display: inline; padding: 10px; }",
+  }, {
+    code: "a { display: inline; margin-left: 10px; }",
+  }, {
+    code: "a { display: inline; margin-right: 10px; }",
+  }, {
+    code: "a { display: inline; vertical-align: baseline; }",
+  }, {
+    code: "a { display: inline-block; }",
+  }, {
+    code: "a { display: inline-block; color: red; }",
+  }, {
+    code: "a { display: inline-block; width: 10px; }",
+  }, {
+    code: "a { display: inline-block; height: 10px; }",
+  }, {
+    code: "a { display: inline-block; margin: 10px; }",
+  }, {
+    code: "a { display: inline-block; padding: 10px; }",
+  }, {
+    code: "a { display: inline-block; vertical-align: baseline; }",
+  }, {
+    code: "a { display: block; }",
+  }, {
+    code: "a { display: block; color: red; }",
+  }, {
+    code: "a { display: block; width: 10px; }",
+  }, {
+    code: "a { display: block; height: 10px; }",
+  }, {
+    code: "a { display: block; margin: 10px; }",
+  }, {
+    code: "a { display: block; padding: 10px; }",
+  }, {
+    code: "a { display: block; float: left; }",
+  }, {
+    code: "a { display: table-cell; width: 100%; }",
+  }, {
+    code: "a { display: table-cell; height: 100%; }",
+  }, {
+    code: "a { display: table-cell; padding: 10px; }",
+  }, {
+    code: "a { position: static; display: block; }",
+  }, {
+    code: "a { position: static; width: 100%; }",
+  }, {
+    code: "a { position: absolute; top: 0px; }",
+  }, {
+    code: "a { position: absolute; right: 0px; }",
+  }, {
+    code: "a { position: absolute; bottom: 0px; }",
+  }, {
+    code: "a { position: absolute; left: 0px; }",
+  }, {
+    code: "a { position: fixed; top: 0px; }",
+  }, {
+    code: "a { position: fixed; right: 0px; }",
+  }, {
+    code: "a { position: fixed; bottom: 0px; }",
+  }, {
+    code: "a { position: fixed; left: 0px; }",
+  } ],
+
+  reject: [ {
+    code: "a { display: inline; width: 100px; }",
```

Comment:
Because I think these tests are already nice and extensive and will only get more extensive as more ignore-pairs are uncovered, Can we add some comments to the tests just to categorize them and make them a little easier to navigate? For example, this section could have the comment `// display: inline rules out width, height, margin-top and margin-bottom, and float`; and for the next section `// display: inline-block rules out ...`